server/controllers/followersController.js

```diff
@@ -136,7 +143,19 @@ function requestFollowers(accessToken, userId) {
             Authorization: `Bearer ${accessToken}`
         }
         // }).then(response => response.data.profiles)
-    }).then(response => response.data.profiles.splice(0, 3)) // Debug
+    }).then(response => {
+        return response.data.profiles.splice(0, 3) // Debug
+    }).then(followers => {
+        return followers.map(follower => {
```

Comment:
It would be better to merge follower and unfollower entities in Backend if we consider as same in a business manner 👍